modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -219,12 +219,12 @@ When Mule clusters are used to serve TCP requests (where TCP includes SSL/TLS, U
 
 There are also many hardware load balancers that can route both TCP and HTTP(S) traffic.
```

Comment:
This just caught my eye, and since I so rarely have much to mark in your super-clean requests, I will mention an extra tidbit from the style ref. here:

“there is” and “there are” at the beginning of a sentence
Sentences that begin with “There is” and “There are” are examples of expletive construction. Because such phrasing tends to bury the actor and action of the sentence, consider rewording.

In this case, you could use something like "Many hardware load balancers can also route both TCP and HTTP or HTTPS traffic" or "You can also use any of several hardware load balancers that can route...."